lib/Intlc/Linter.hs

```diff
@@ -1,16 +1,39 @@
 module Intlc.Linter where
+import           Data.Char (isAscii)
+import qualified Data.Text as T
 
 import           Intlc.ICU
 import           Prelude   hiding (Type)
 
+
+import           Text.Show
 data LintingError
   = TooManyInterpolations
-  deriving (Eq, Show)
+  | InvalidNonAsciiCharacter (NonEmpty Char)
+  deriving (Eq)
+
+instance Show LintingError where
+  show TooManyInterpolations = "Nested functions not allowed"
+  show (InvalidNonAsciiCharacter chars)   =  "These non-ascii characters are not allowed: " <> Prelude.show (toList chars)
```

Comment:
I think we can fix Unicode in stdout by dropping `Prelude.show` here. :slightly_smiling_face:

Reply:
Ah, I investigated how to fix that issue but gave up and was going to ask you for help. Thanks!